Applications/MaterialAssist/Client/MaterialAssistClientBoards.cs

```diff
@@ -0,0 +1,196 @@
+
+        /// <inheritdoc />
+        public async Task<IEnumerable<BoardEntity>> GetBoardEntitiesByBoardCode(string boardCode)
+        {
+            var url = $"{_BaseRoute}?{_BoardCode}={Uri.EscapeDataString(boardCode)}";
+
+            return await RequestEnumerable<BoardEntity>(new Uri(url, UriKind.Relative));
+        }
+
+        /// <inheritdoc />
+        public async Task<IEnumerable<BoardEntity>> GetBoardEntitiesByBoardCodes(IEnumerable<string> boardCodes)
+        {
+            var url = $"{_BaseRoute}";
+
+            var appendedBoardCodes = new StringBuilder("?");
+            appendedBoardCodes.Append(string.Join("&", boardCodes.Select(boardCode => $"{_BoardCode}={Uri.EscapeDataString(boardCode)}")));
+
+            url += boardCodes;
+
+            return await RequestEnumerable<BoardEntity>(new Uri(url, UriKind.Relative));
+        }
+
+        /// <inheritdoc />
+        public async Task<IEnumerable<StorageLocation>> GetStorageLocations()
+        {
+            var url = $"{_BaseRoute}/storageLocations";
+
+            return await RequestEnumerable<StorageLocation>(new Uri(url, UriKind.Relative));
+        }
+
+        #endregion Read
+
+        #region Create
+
+        /// <inheritdoc />
+        public async Task<BoardEntity> CreateBoardEntity(string boardTypeCode, BoardEntity boardEntity)
+        {
+            var url =
+                $"{_BaseRoute}?{_BoardTypeCode}={Uri.EscapeDataString(boardTypeCode)}";
+
+            return await PostObject(new Uri(url, UriKind.Relative), boardEntity);
+        }
+
+        /// <inheritdoc />
+        public async Task CreateBoardType(BoardType boardType)
+        {
+            var url = $"{_BaseRouteMaterialManager}";
+
+            await PostObject(new Uri(url, UriKind.Relative), boardType);
+        }
+
+        #endregion Create
+
+        #region Update
+
+        /// <inheritdoc />
+        public async Task UpdateBoardEntityDimensions(string id, double length, double width)
+        {
+            var url = $"{_BaseRoute}/{Uri.EscapeDataString(id)}?{_Length}={length}&{_Width}={width}";
+
+            await PatchObject(new Uri(url, UriKind.Relative));
+        }
+
+        /// <inheritdoc />
+        public async Task UpdateBoardEntityComments(string id, string comments)
+        {
+            var url = $"{_BaseRoute}/{Uri.EscapeDataString(id)}?{_Comments}={Uri.EscapeDataString(comments)}";
+
+            await PatchObject(new Uri(url, UriKind.Relative));
+        }
+
+        /// <inheritdoc />
+        public async Task StoreBoardEntity(string id, int length, int width, StorageLocation storageLocation)
+        {
+            var url = $"{_BaseRoute}/{Uri.EscapeDataString(id)}/store?{_Length}={length}&{_Width}={width}&{_StorageLocation}={storageLocation}";
+
+            await PatchObject(new Uri(url, UriKind.Relative));
+        }
+
+        /// <inheritdoc />
+        public async Task RemoveAllBoardEntities(string id, bool deleteBoardFromInventory = false)
```

Comment:
Remove is missleading. The term nees to get discussed. RemoveFromWorkstation might be an option. Also pick.

Reply:
is this a blocker for the concept? can´t we also allign it when doing the actual implementation?

Reply:
RemoveFromWorkstation would be good. We changed the term from "pick" to "remove" in the app.

Reply:
Should it be named RemoveAllBoardEntitiesFromWorkstation as we have 3 methods there to remove a Subset and also a single board, same for edgebands